providers/burstsms/src/lib/burstSms.provider.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+import axios, { AxiosInstance } from 'axios';
+import qs from 'qs';
+
+export class BurstSmsSmsProvider implements ISmsProvider {
```

Comment:
```suggestion
export class BurstSmsProvider implements ISmsProvider {
```
We do not need to add the type of provider in the name as we can clearly see it from channel property and thus we avoid the cacophony.